static/html/mozorg/ghostery.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
I assume these will only show up in English promotions since there is no l10n here?